WordPressKit/WordPressComRestApi.swift

```diff
@@ -44,8 +44,8 @@ open class WordPressComRestApi: NSObject {
     public typealias SuccessResponseBlock = (_ responseObject: AnyObject, _ httpResponse: HTTPURLResponse?) -> ()
     public typealias FailureReponseBlock = (_ error: NSError, _ httpResponse: HTTPURLResponse?) -> ()
 
-    @objc public static let apiBaseURLString: String = "https://public-api.wordpress.com/"
-    
+    @objc public static let BaseURLStringDefault = "https://public-api.wordpress.com"
```

Comment:
should this be with a lower case b? "baseURLStringDefault"?

Reply:
I'll admit I'm not super familiar with convention here. This matches the other static constants here like `LocaleKeyDefault`, `LocaleKeyV2` and `SessionTaskKey`.

Happy to change to `baseURLStringDefault` if that is more correct though.

Reply:
Yeah its a mix there, ah? defaultBackgroundSessionIdentifier is lower case and so was apiBaseURLString. To me lower case makes more sense.

Reply:
Lower case is the standard on Swift.

Reply:
👋 Hello there. I agree that the case conventions here are consistent; the majority of them are capitalized. Should you decide to make this change, please keep in mind the visibility of these properties (i.e., `public`), and the subsequent semantic version impact of making them lowercase.

Regarding naming conventions, at the moment our [Swift Style Guide](https://github.com/wordpress-mobile/swift-style-guide) does not express an opinion, but I tend to prefer "Pascal case" for types & protocols, and "lower camel case" for everything else. If we proceed with the change you're proposing, perhaps we could also amend the style guide to reflect this new precedent? We could then enable [this](https://github.com/realm/SwiftLint/blob/master/Rules.md#type-name) SwiftLint rule.

Reply:
Thanks for the input folks. I’ll go with the lower camel case as suggested. My day to day exposure to Swift is pretty low so I’m happy to lean on your feedback.

I’m aware of the versioning implications here. As is, this is clearly a breaking change. I could make the API changes additive only by keeping this constant as `apiBaseUrlString` but that naming is a little misleading since it’s now just the default url.

Reply:
Although maybe `apiBaseUrlString` isn’t such a bad name 🤔 It is still a constant with the real WP.com endpoint.